From cdaea6e3002ea138a7c7d2b66b4f3ae290d85df7 Mon Sep 17 00:00:00 2001
From: Ikey Doherty <ikey@solus-project.com>
Date: Fri, 16 Dec 2016 15:13:51 +0000
Subject: [PATCH] Disable -Werror=array-bounds

X.Org Server 1.19.0 fails to compile with optimisations when this error
flag is set, turning warnings fatal. Revert the behaviour.

Signed-off-by: Ikey Doherty <ikey@solus-project.com>
---
 aclocal.m4 | 1 -
 1 file changed, 1 deletion(-)

diff --git a/aclocal.m4 b/aclocal.m4
index 1a4ca23..70a2a69 100644
--- a/aclocal.m4
+++ b/aclocal.m4
@@ -3360,7 +3360,6 @@ XORG_TESTSET_CFLAG([[BASE_]PREFIX[FLAGS]], [-Werror=missing-braces])
 XORG_TESTSET_CFLAG([[BASE_]PREFIX[FLAGS]], [-Werror=sequence-point])
 XORG_TESTSET_CFLAG([[BASE_]PREFIX[FLAGS]], [-Werror=return-type], [-errwarn=E_FUNC_HAS_NO_RETURN_STMT])
 XORG_TESTSET_CFLAG([[BASE_]PREFIX[FLAGS]], [-Werror=trigraphs])
-XORG_TESTSET_CFLAG([[BASE_]PREFIX[FLAGS]], [-Werror=array-bounds])
 XORG_TESTSET_CFLAG([[BASE_]PREFIX[FLAGS]], [-Werror=write-strings])
 XORG_TESTSET_CFLAG([[BASE_]PREFIX[FLAGS]], [-Werror=address])
 XORG_TESTSET_CFLAG([[BASE_]PREFIX[FLAGS]], [-Werror=int-to-pointer-cast], [-errwarn=E_BAD_PTR_INT_COMBINATION])
-- 
2.11.0

